source/beamline/AMPVControl.h

```diff
@@ -132,12 +132,16 @@ class AMReadOnlyPVControl : public AMControl {
 
 protected slots:
 	/// Handles the low limit PV changing. Converts the limit before forwarding the signal on.
-	virtual void onLowLimitPVValueChanged(double newLowLimit);
+	virtual void setLowLimitValue(double newLowLimit);
```

Comment:
The implementation isn't different, but the usage is. Previously, the values were changed in `onWritePVInitialized` type slots, without the signals being emitted. Calling `onLowLimitPVValueChanged` in these cases isn't really representative of what's happening.